transitionmonitor_docker/Dockerfile

```diff
@@ -118,9 +118,9 @@ RUN Rscript -e "\
 # copy in PACTA repos
 COPY pacta-data /pacta-data
 COPY r2dii.stress.test.data /r2dii.stress.test.data
-COPY PACTA_analysis /bound
-COPY create_interactive_report /create_interactive_report
 COPY r2dii.climate.stress.test /r2dii.climate.stress.test
+COPY create_interactive_report /create_interactive_report
+COPY PACTA_analysis /bound
```

Comment:
@cjyetman since these are the most active, I put them at the end, hoping to cache the other 3.